db/seeds.rb

```diff
@@ -1,7 +1,230 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+include SampleData
+require "faker"
+
+Rails.cache.clear
+
```

Comment:
Sorry that we don't have a Linter in place... but can you please be sure that you're using two spaces for indentation?

Reply:
for some reason my vscode was setup to indent four spaces 🙃 fixed it!